spec/services/activity/import/field_spec.rb

```diff
@@ -0,0 +1,377 @@
+RSpec.describe Activity::Import::Field do
+  describe ".all" do
+    subject { described_class.all }
+
+    it "returns an array of all Field objects" do
+      expect(subject.count).to eq 50
+      expect(subject).to all(be_a(described_class))
```

Comment:
I have _opinions_ on the readability of `described_class` in general, but for this type-checking tests i.e. `to be_a(Field)` I prefer explicitly asserting on the name of the class as I find it more readable in the context of an individual test. Is there something `described_class` gives us over the use of the class name specifically, or is more of a convention?

Reply:
I used to work on a project with a very strict Rubocop, so I've picked up some habits from that. No particular benefit, happy to change it.